pkg/analyze/host_cpu.go

```diff
@@ -63,11 +63,21 @@ func (a *AnalyzeHostCPU) Analyze(
 
 	results, err := analyzeHostCollectorResults(collectedContents, a.hostAnalyzer.Outcomes, a.CheckCondition, a.Title())
 	if err != nil {
-		return nil, errors.Wrap(err, "failed to analyze OS version")
+		return nil, errors.Wrap(err, "failed to analyze CPU info")
 	}
 
-	return results, nil
+	// Add template support
+	content := collectedContents[0].Data
+	cpuInfo := collect.CPUInfo{}
+	if err := json.Unmarshal(content, &cpuInfo); err != nil {
+		return nil, errors.Wrap(err, "failed to unmarshal cpu info")
+	}
 
+	for _, r := range results {
+		r.Message = strings.ReplaceAll(r.Message, "{{ Info.MachineArch }}", cpuInfo.MachineArch)
+	}
```

Comment:
Would it be possible to actually template this instead of running a strings.ReplaceAll? I really hope that isn't a pattern in the rest of the project but if it is it's not one I'd really prefer to continue...